Libplanet/Action/ActionEvaluator.cs

```diff
@@ -40,15 +41,19 @@ public class ActionEvaluator<T>
         /// the states for a provided <see cref="Address"/>.</param>
         /// <param name="trieGetter">The function to retrieve a trie for
         /// a provided <see cref="BlockHash"/>.</param>
+        /// <param name="genesisHash"> A <see cref="HashDigest{SHA256}"/> value of the genesis block
```

Comment:
It doesn't seem a `HashDigest<SHA256>`:

```suggestion
        /// <param name="genesisHash">A <see cref="BlockHash"/> value of the genesis block
```